src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputRightCurly.java

```diff
@@ -1,9 +1,11 @@
 package com.google.checkstyle.test.chapter4formatting.rule412nonemptyblocks;
 
-public class InputRightCurly2 {
+public class InputRightCurly {
     public static void main(String[] args) {
         boolean after = false;
         try {
-        } finally { after = true; }
+        } finally {
```

Comment:
I noticed we changed original formatting in input files. This is just one example, there others. Considering it doesn't affect existing violations under the test, I'm ok with the changes. However, in next PRs I suggest to not mix formatting with migration. cc @romani

Reply:
@Zopsss , please restore, in this Check, such formatting matters.

Reply:
@Zopsss , I merged to move forward big changes, please send separate PR to restore location of curly brace

Reply:
It was a violation that's why I had to change it. Anyways I've restored it here: https://github.com/checkstyle/checkstyle/pull/15148#issue-2377311149